docs/docs/schema.md

```diff
@@ -0,0 +1,367 @@
+---
+
+title: Schema
+description: Define the schema of your database.
+
+---
+
+## Overview
+
+In Drift, your schema is represented as a table. Each column in the table represents a field in your data class.
```

Comment:
Pedantically speaking, the schema is actually represented by multiple tables (and views, indices, ...) :nerd_face: 

Perhaps we can phrase this introduction as a "Drift is a library for relational databases, meaning that your data is structured into data with predefined columns" or something?
The rest of the introduction is nice :+1: